src/common-components/AlertDismissible.jsx

```diff
@@ -8,8 +8,7 @@ function AlertDismissible(props) {
   const [show, setShow] = useState(true);
 
   return show ? (
-    <Alert variant={props.variant} onClose={() => setShow(false)} dismissible className="pb-1 pt-1">
-      <Icon src={Info} className="alert-icon" />
+    <Alert variant={props.variant} onClose={() => setShow(false)} dismissible className="pb-1 pt-1" icon={Info}>
```

Comment:
use `Error ` instead of `Info` icon like using in other components.

Reply:
I think variant may be different for different components that's why we are using `Info`